server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -256,7 +257,7 @@ func (s *BuildBuddyServer) CreateGroup(ctx context.Context, req *grpb.CreateGrou
 		Name:                   groupName,
 		OwnedDomain:            groupOwnedDomain,
 		SharingEnabled:         req.GetSharingEnabled(),
-		UseGroupOwnedExecutors: req.GetUseGroupOwnedExecutors(),
+		UseGroupOwnedExecutors: sql.NullBool{Valid: true, Bool: req.GetUseGroupOwnedExecutors()},
```

Comment:
To me it seems slightly error prone that we have to provide `Valid: true` when using these null types, although it is more convenient to get the values out of them (no need to check `!= nil` which is nice).

What do people think about using raw pointers instead? I think those are slightly error prone as well (it's possible to accidentally de-reference `nil` pointers) but I wonder if checking `nil` before de-referencing is more instinctual than remembering to pass `Valid: true`.